README.md

```diff
@@ -18,11 +18,14 @@
    1. [Kernel Messaging](#kernel-messaging)
    1. [Kernel Output](#kernel-output)
    1. [Launcher](#launcher)
+   1. [Leap Counter (Reusability 1C)](#leap-counter-reusability-1c)
    1. [Log Messages](#log-messages)
    1. [Main Menu](#main-menu)
    1. [Metadata Form](#metadata-form)
    1. [MIME Renderer](#mime-renderer)
    1. [Notifications](#notifications)
+   1. [Step Counter (Reusability 1A)](#step-counter-reusability-1a)
+   1. [Step Counter Extension (Reusability 1B)](#step-counter-extension-reusability-1b)
```

Comment:
What if it was named:
- Reusability part 1 - step counter
- Reusability part 2 - step counter extension
- Reusability part 3 - log counter

There are three changes I am suggesting feel free to only take some of them:
- using the word "part" clears the question "what does 1c means"
- swapping the order puts emphasis on Reusability and brings the items together on the list; most folks are probably less interested in implementing a step counter/leap counter and more on learning about reusability
- dropping a/b/c in favour of just numerals may be easier to parse especially if there are no 2a/2b/2c (Arabic numerals are slightly more universal than English alphabet).